### PR TITLE
Heat Recovery unit entity type

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 1]
```diff
@@ -17,6 +17,14 @@
 ########################
 
 ### Single Zone Units ###
+AHU_SS_SDM:
+  guid: "d289ff35-4b60-42ba-b32e-00a679add978"
+  description: "AHU with supply air damper monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - SS
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:26]: This is discouraged as it is too vague. Please use SFSS, DFSS (whichever is more appropriate)

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 2]
```diff
@@ -17,6 +17,14 @@
 ########################
 
 ### Single Zone Units ###
+AHU_SS_SDM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:20]: This is not a complete type. For AHU, we expect fan types, cooling/heating types, economizer types, etc.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml:20]: @cstirdivant - thanks for your response. it is a heat recovery unit - It has the following points from the BMS commissioning engineers - 
run_command
supply_air_damper_command
run_command
. Can you provide me with any guidance. Thank you

**Comment by Ciaran-Oflannagain** [comment]
hi @cstirdivant - just following up on the above PR. Many thanks, Ciaran

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: @ccquigley - apologies have received updated info from commissioning engineers. I have remodelled this as a FCU. It is a standalone unit that exchanges air using the damper.

**Comment by Ciaran-Oflannagain** [comment]
@ccquigley - apologies have received updated info from commissioning engineers. I have remodelled this as a FCU. It is a standalone unit that exchanges air using the damper.

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 3]
```diff
@@ -1038,6 +1038,15 @@ FCU_SS_ZTM:
   - SS
   - ZTM
 
+FCU_SS_SDM:
```
Comment (ccquigley) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1041]: Does this FCU have a fan? Just trying to see if this equipment could fall under an existing FCU type but most existing types include fan points.

**Review by ccquigley — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 3 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1041]: @ccquigley - Unfortunately no FANs on it. Just the SS and supply damper monitoring points to exchange the air. Thank you, Ciaran

Comment (ccquigley) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1051]: Can we get more information on what the start stop is for then? Is there a fan but we don't have points or is this for heating/cooling? Also is this a landlord unit?

**Comment by Ciaran-Oflannagain** [comment]
@ccquigley - upon review, the damper monitoring points have been omitted so there is a prexisting entity type hereworks can utilise. Closing PR. Thanks for your assistance

**Pull request closed without merging**